server/objects/panel.js

```diff
@@ -25,6 +25,46 @@
     },
   };
 
+  module.exports.addPanelMembersDAO = {
+    getStatus: function(app, req, trialNumber, courtLocationCode) {
+      const payload = {
+        uri: urljoin(
+          config.apiEndpoint,
+          `moj/trial/panel/status?trial_number=${trialNumber}&court_location_code=${courtLocationCode}`,
+        ),
+        method: 'GET',
+        headers: {
+          'User-Agent': 'Request-Promise',
+          'Content-Type': 'application/vnd.api+json',
+          Authorization: req.session.authToken,
+        },
+        json: true,
+      };
+
+      app.logger.info('Sending request to API: ', payload);
+
+      return rp(payload);
+    },
+
+    post: function(app, req, body) {
+      const payload = {
+        uri: urljoin(config.apiEndpoint, 'moj/trial/panel/add-panel-members'),
+        method: 'POST',
+        headers: {
+          'User-Agent': 'Request-Promise',
+          'Content-Type': 'application/vnd.api+json',
+          Authorization: req.session.authToken,
+        },
+        json: true,
+        body,
+      };
+
+      app.logger.info('Sending request to API: ', payload);
+
+      return rp(payload);
+    },
+  };
+
```

Comment:
these should use the new DAO() class constructor available in `server/objects/dataAccessObject`

Reply:
Changed